### PR TITLE
3405915065 migrate weld models

### PR DESCRIPTION
**Pull request by alewinskylyka**
Hey Ana, there is a lot in this and its a bit of a mess so let's discuss

### CODE REVIEW

File: models/staging/portal/stg_portal_pricing_model.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+-- timestamps
+
+-- attributes
+        name,
+        description,
+
+-- metadata
+        _fivetran_synced as fivetran_sync_datetime,
+
```
Comment (agpmercado) [inline models/staging/portal/stg_portal_pricing_model.sql:21]: Add dbt_update_datetime

File: models/staging/portal/stg_portal_shipping_suburb_postcode.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+-- attributes
+        -- notes,
+--TIM
+        onboarding_status as onboarding_status_flag,
+        postcode,
+        suburb,
+
+-- metadata
+        _fivetran_synced as fivetran_sync_datetime
+
```
Comment (agpmercado) [inline models/staging/portal/stg_portal_shipping_suburb_postcode.sql:28]: Add dbt_update_datetime

File: models/staging/portal/stg_portal_subcription.sql [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+-- attributes
+        cancelled_date as cancel_date,
+        charge_day as charge_day_of_week,
+        charge_frequency as charge_frequency_rate,
+        frequency as delivery_frequency_rate,
+        current_price as current_price_amount,
+        current_price_per_kilo as current_price_per_kilo_amount,
+        formula_price_per_kilo as formula_price_per_kilo_amount,
+        custom_delivery_day as custom_delivery_day_of_week,
```
Comment (agpmercado) [inline models/staging/portal/stg_portal_subcription.sql:26]: end naming convention should be date or is this a flag?

File: models/staging/portal/stg_portal_user.sql [hunk 4]
```diff
@@ -39,7 +39,7 @@ renamed as (
         number_of_charges as number_of_charges_count,
         bab_version as bab_version_flag,
 
---metadata
+-- metadata
         _fivetran_synced as fivetran_sync_datetime,
```
Comment (agpmercado) [inline models/staging/portal/stg_portal_user.sql:43]: Add dbt_update_datetime

File: models/staging/portal/stg_portal_user_type.sql [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    select
+-- ids
+        id as user_type_id,
+
+-- attributes
+        name as user_type_name,
+
+-- metadata
+        _fivetran_synced as fivetran_sync_datetime,
+
```
Comment (agpmercado) [inline models/staging/portal/stg_portal_user_type.sql:18]: add dbt_update_datetime

File: models/staging/portal/stg_portal_pricing_model.sql [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+renamed as (
+
+    select
+-- ids
+        id as pricing_model_id,
+
+-- timestamps
+
+-- attributes
+        name,
```
Comment (agpmercado) [inline models/staging/portal/stg_portal_pricing_model.sql:16]: does this need to be more descriptive? like subscription_model_name ?

File: models/staging/portal/stg_portal_subcription.sql (hunk 3 above)
Comment (alewinskylyka) [reply models/staging/portal/stg_portal_subcription.sql:28]: I've updated to customer_delivery_weekday

Comment (alewinskylyka) [reply models/staging/portal/stg_portal_subcription.sql:28]: We can chat to Oscar to see what he thinks here although fairly minor issue

File: models/staging/portal/stg_portal_pricing_model.sql (hunk 6 above)
Comment (alewinskylyka) [reply models/staging/portal/stg_portal_pricing_model.sql:16]: Good pick up. Do you suggest the description field name should also be updated to pricing_model_description in that case?

**Pull request merged**